std/machines/hash/keccakf.asm

```diff
@@ -0,0 +1,8 @@
+// TODO: placeholder for Rust wrapper
+machine KeccakF with
+    latch: LATCH,
+{ 
+    let x;
+    let y;
+    operation keccakf x, y ->;
+}
```

Comment:
The operation should really be `x, y, STEP ->`. Not sure how to "import" the `STEP` fixed column from the main machine here.

Doesn't really matter at this point though given it's a placeholder.

Reply:
Done.